tools/schema/Opc.Ua.NodeSet2.Reduced.xml

```diff
@@ -1716,7 +1716,3116 @@
       <Reference ReferenceType="HasTypeDefinition">i=72</Reference>
     </References>
     <Value>
-    <ByteString xmlns="http://opcfoundation.org/UA/2008/02/Types.xsd"></ByteString>
+    <ByteString xmlns="http://opcfoundation.org/UA/2008/02/Types.xsd">PG9wYzpUeXBlRGljdGlvbmFyeQ0KICB4bWxuczpvcGM9Imh0dHA6Ly9vcGNmb3VuZGF0aW9uLm9y
+Zy9CaW5hcnlTY2hlbWEvIg0KICB4bWxuczp4c2k9Imh0dHA6Ly93d3cudzMub3JnLzIwMDEvWE1M
```

Comment:
Would it be also an option that we don't change the reduced nodeset and include insted the ns0-bsd where needed?